src/dune_rules/module_compilation.ml

```diff
@@ -486,7 +486,8 @@ let build_all cctx =
             if Modules.is_stdlib_alias modules m then
               (* XXX it would probably be simpler if the flags were just for this
                  module in the definition of the stanza *)
-              Compilation_context.for_alias_module cctx m
+              let flags = Compilation_context.flags cctx in
+              Compilation_context.for_alias_module cctx m ~flags
```

Comment:
Seems weird to pass a flag to Compilation_context that's coming from the Compilation_context